src/main/java/com/hibernate/cinema/dao/MovieDao.java

```diff
@@ -6,5 +6,7 @@
 public interface MovieDao {
     Movie add(Movie movie);
 
+    Movie get(Long id);
```

Comment:
point 2 of the check list: Do not create redundant functionality, like methods, that are not required by task

Reply:
I understand it, but not sure how can I get a movie by id without this method